api/src/main/java/org/openmrs/module/reporting/common/SqlRunner.java

```diff
@@ -204,7 +205,7 @@ else if (paramValue instanceof OpenmrsObject) {
                         sqlVal = ((OpenmrsObject)paramValue).getId().toString();
                     }
                     else {
-                        sqlVal = "'" + paramValue.toString() + "'";
+                        sqlVal = "'" + StringEscapeUtils.escapeSql(paramValue.toString()) + "'";
```

Comment:
I just added it the escape here because I believe all the other cases are non-strings?